form-ics213.html

```diff
@@ -14,6 +14,14 @@
   <body>
     <div class="version">0.1.0</div>
```

Comment:
Time to increment the version number (here and in form-oa-muni-status.html)?

Reply:
Good point. I'll add a commit to address this shortly.